pubsub/mqtt/mqtt.go

```diff
@@ -336,6 +333,8 @@ func (m *mqttPubSub) connect(ctx context.Context, clientID string) (mqtt.Client,
 		return nil, err
 	}
 	opts := m.createClientOptions(uri, clientID)
+	// Turn off auto-ack
+	opts.SetAutoAckDisabled(true)
```

Comment:
Does this need to be changed in master too?

Reply:
It is already there in master.